src/PSRule.Rules.Azure/rules/Azure.VM.Rule.yaml

```
@@ -110,6 +110,24 @@ spec:
     field: properties.dnsSettings.dnsServers
     hasValue: false
 
+---
+# Synopsis: VMs should be deallocated instead of stopped.
+apiVersion: github.com/microsoft/PSRule/v1
+kind: Rule
+metadata:
+  name: Azure.VM.ShouldNotBeStopped
+  ref: AZR-123346
```

Use the next available ref based on the bottom of this table. https://azure.github.io/PSRule.Rules.Azure/en/rules/

```suggestion
  ref: AZR-000351
```